apps/docs/src/guide/transactions/snippets/transaction-request/add-witness.ts

```diff
@@ -16,23 +16,23 @@ const transactionRequest = new ScriptTransactionRequest({
   script: ScriptSum.bytecode,
 });
 
-// #region transaction-request-6
+// #region transaction-request-10
 // Add a witness directly
 transactionRequest.addWitness(witness);
```

Comment:
Is it worth showing a signature as the witness here?

Maybe generated via `account.signTransaction(transactionRequest)`

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3440/commits/b94fb0a0e783e8a1a467a92e89bf870cfefb9f6d

Reply:
I was thinking both side by side:
```ts
// Add a witness signature directly
const signature = account.signTransaction(transactionRequest);
transactionRequest.addWitness(witness);

// Or add multiple via `addAccountWitnesses`
await transactionRequest.addAccountWitnesses([accountA, accountB]);
```

Reply:
@petertonysmith94 seems reasonable https://github.com/FuelLabs/fuels-ts/pull/3440/commits/a847cdb43931e1691218cfa9904f1402da158e16